docs/quickstart.rst

```diff
@@ -145,18 +145,25 @@ Note that this is not the same thing as::
   $ BORG_PASSPHRASE='complicated & long'; # missing "export"!
   $ ./yourborgjob.sh # will never see the above $BORG_PASSPHRASE value
 
+This is because ``export`` exposes variables to subprocesses, which borg may be
+one of. More on ``export`` can be found in the "ENVIRONMENT" section of the
+bash(1) man page, or the relevant POSIX_ spec.
+
+.. _POSIX: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_22.
+
```

Comment:
keep the bash(1) reference, but if someone wants more, he can google the link to the posix spec by himself.